### PR TITLE
Add Joepegs avalanche NFT trades - Ready for review 

### PR DESCRIPTION
**Pull request by henrystats**

### CODE REVIEW

**Comment by 0xRobin** [comment]
marked this `WIP` as the CI is failing on a uniqueness test.

**Comment by henrystats** [comment]
thank you ser, will fix this shortly!

**Comment by henrystats** [comment]
hola ser @0xRobin this is now ready for review!

File: models/_sector/nft/trades/chains/avalanche_c/platforms/_schema.yml [hunk 1]
```diff
@@ -77,4 +77,110 @@ models:
             - seller
             - nft_contract_address
             - nft_token_id
-            - price_raw
\ No newline at end of file
+            - price_rawversion: 2
```
Comment (0xRobin) [inline models/_sector/nft/trades/chains/avalanche_c/platforms/_schema.yml]:
```suggestion
            - price_raw
```
copy/paste artifact

File: models/_sector/nft/trades/chains/avalanche_c/platforms/_schema.yml [hunk 2]
```diff
@@ -77,4 +77,110 @@ models:
...
+          match_columns:
+            - block_number
+            - tx_hash
+            - sub_tx_trade_id
+          check_columns:
+            - buyer
+            - seller
+            - nft_contract_address
+            - nft_token_id
+            - price_raw
```
Comment (0xRobin) [inline models/_sector/nft/trades/chains/avalanche_c/platforms/_schema.yml]: looks like you copy/pasted the existing definitions as well

**Review by 0xRobin — APPROVED** [review]

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**